src/components/shared/Entity/ValidationErrorSummary/DetailsErrors.tsx

```diff
@@ -0,0 +1,47 @@
+import KeyValueList from 'components/shared/KeyValueList';
+import { useRouteStore } from 'hooks/useRouteStore';
+import { useIntl } from 'react-intl';
+import { entityCreateStoreSelectors } from 'stores/Create';
+import { hasLength } from 'utils/misc-utils';
+
+function DetailsErrors() {
```

Comment:
Broke up the errors to be in their own components. This is because we want to move away from having all the errors in a summary at the top. I put some work into that but the styling was a bit off and I didn't want to make this PR wait for that functionality.